@xen-orchestra/fs/src/local.js

```diff
@@ -114,7 +122,11 @@ export default class LocalHandler extends RemoteHandlerAbstract {
   }
 
   async _unlink(file) {
-    return fs.unlink(this._getFilePath(file))
+    const filePath = this._getFilePath(file)
+    return await retry(() => fs.unlink(filePath), {
```

Comment:
We don't need to wait for the promise.

Reply:
Without awaiting the response, the client was with the error : `Expected values to be strictly equal:\n\n0 !== 1536\n`.

Adding `wait` fixed this issue.